### ISSUE TITLE
Question: How the same metadata is inserted into each other's signals.

### ISSUE THREAD

**Issue by chenmudu** [pyroscope-io/pyroscope-java#69]
Hi, bro, thanks very much for these excellent products, but I have some troubles about using them.
I used the following tools:
- otel-java-agent  v1.17.0
- pyroscope-java v0.10.1
- otel-profiling-java v0.10.1.3.jar 
- my-java-service-app ignore version.

I'm trying to understand what's going on internally, I can see that spans is being processed, put in the span id named profile_id, and a few others as seen in the Root Span in Jaeger UI, for example: pyroscope.profile.baseline.url, pyroscope.profile.url and many more in the **otel-profiling-java**. 

But when I tried to use pyroscope ui, I found additional search conditions, such as span name in select tag, profile_id can be searched directly in the search box, but I didn't find how to combine span_name and profiler_id and maybe some other metadata in the profile signal, which puzzles me quite a bit.

To sum up, there are two points: 
1. How to put span info into the current root span. (I already know how it works) 
2. How to put span info into the current profile signal (**still confused**).

I'm trying to figure out how to put span info inside profile signal, but I can't find any entry. I can see how the prometheus exemplars are placed inside the metrics labels. Maybe I'm doing it wrong and I'd like some help. thank you very much.♥

**Comment by chenmudu** [comment]
I will close this issue after getting an official answer.

**Comment by qsLI** [comment]
> I will close this issue after getting an official answer.

I am confused too. Do you get official answer?
